spec/rubocop/ast/send_node_spec.rb

```diff
@@ -94,6 +94,35 @@
     end
   end
 
+  describe '#access_modifier?' do
+    context 'when inside a module' do
+      let(:send_nodes_in_module) { source.ast.children[1].children }
+
+      let(:source) { parse_source(<<-RUBY) }
+        module Foo
+          module_function
+          public
+          protected
+          private
+          custom_macro!
+          self.private
+        end
+      RUBY
+
+      it 'classifies specific macros like `private` as access modifiers' do
```

Comment:
I think the RSpec core guys view short non-descriptive specs as no longer a good thing. Hence them removing the bare ` it { should }` and recommending you not use `is_expected.to`.

That is just a note though. I'll almost always support in-file consistency over other high level things like that so will change :smile: